course/lesson-22-strings/lesson.tres

```diff
@@ -0,0 +1,198 @@
+title = "What are strings"
+type = 0
+text = "A string is a variable type which holds text instead of numbers. When we use strings, we use quotation marks (\"\") to differentiate them from other variable types, or method names for example.
+
+You may remember we've used strings before in previous lessons."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=6]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-S5AJTESv.tres"
+title = ""
+type = 0
+text = ""
+visual_element_path = "visuals/PrintString.tscn"
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=7]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-5tT6n1Tx.tres"
+title = ""
+type = 0
+text = "Strings are like an array of characters. In fact, we can even loop through the characters of a string in the same way."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=8]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-lTtcyhcI.tres"
+title = ""
+type = 0
+text = ""
+visual_element_path = "visuals/PrintCharactersInName.tscn"
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=4]
+script = ExtResource( 2 )
+quiz_id = "res://course/lesson-22-strings/quiz-yqdYOexR.tres"
+question = "Which of these are strings?"
+content_bbcode = ""
+hint = ""
+explanation_bbcode = "[code]\"1\"[/code] and [code]\"name\"[/code] are strings.
+
+[code]\"1\"[/code] only contains a character and [i]doesn't[/i] represent the number [code]1[/code].
+
+[code]\"name\"[/code] is made up of four different characters."
+answer_options = [ "1", "\"1\"", "\"name\"" ]
+valid_answers = [ "\"1\"", "\"name\"" ]
+is_multiple_choice = true
+do_shuffle_answers = true
+
+[sub_resource type="Resource" id=9]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-b5BnVHAR.tres"
+title = "Why we use strings"
+type = 0
+text = "Strings are very useful in games and applications. We can display information such as the description of a spell, or the name of a character."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=10]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-bBWNbDGt.tres"
+title = ""
+type = 0
+text = ""
+visual_element_path = "visuals/RobotWithRegularName.tscn"
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=11]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-6fUqbTxY.tres"
+title = ""
+type = 0
+text = "Every piece of text you see in this app is a string that Godot is displaying for us.
+
+Much like how [code]Vector2[/code] variables make calculations easier, string variables come with lots of helper methods and tricks we can use.
```

Comment:
We've used the term function throughout the app so far. We're not gonna cover OOP or objects so we can't really talk about methods and properties, which are OO concepts